src/components/table/table-cell/table-cell.component.js

```diff
@@ -19,14 +20,41 @@ import OptionsHelper from '../../../utils/helpers/options-helper';
  * set styling options for the cell used for action such as delete.
  */
 class TableCell extends React.Component {
+  /**
+   * Returns if child element has type property.
+   */
+  childHasType(child) {
+    return child && child.type;
+  }
+
+  /**
+   * Returns if child element is Textarea.
+   */
+  childrenHasTextArea() {
+    const { children } = this.props;
+
+    if (!Array.isArray(children)) {
+      return this.childHasType(children) && children.type.name === TextArea.name;
+    }
+
+    let result = false;
+    children.forEach((child) => {
+      if (this.childHasType(child) && child.type.name === TextArea.name) {
+        result = true;
+      }
+    });
+    return result;
+  }
+
   /**
    * Returns props to be used on the TD element.
    */
   get tableCellProps() {
     const { ...props } = validProps(this);
-
     delete props.children;
     props.align = this.props.align;
+    props.size = this.props.size;
```

Comment:
Aren't align and size already set? If not, is it because they need to be marked as safe when calling validProps? I.e. `validProps(this, ['align', 'size'])`.

Reply:
Yeah good point, somehow forgot that `validProps` had been updated to allow that